lkml/__init__.py

```diff
@@ -117,7 +138,18 @@ def cli():
     logging.getLogger().setLevel(args.log_level)
 
     result: dict = load(args.file)
-    args.file.close()
 
-    json_string = json.dumps(result, indent=2)
-    print(json_string)
+    try:
+        if args.format:
+            args.file.seek(0)
+            dump(result, args.file)
+            args.file.truncate()
+        elif args.lookml:
+            lookml_string = dump(result)
+            print(lookml_string)
+        elif args.json:
+            json_string = json.dumps(result, indent=2)
+            print(json_string)
+
+    finally:
```

Comment:
FileType objects dont work with context managers, this makes sure the file is closed even if they cannot be parsed.